doc/planar.xml

```diff
@@ -0,0 +1,314 @@
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> D := Digraph([[3, 5, 10], [8, 9, 10], [1, 4], [3, 6], 
+> [1, 7, 11], [4, 7], [6, 8], [2, 7], [2, 11], [1, 2], [5, 9]]);
+<digraph with 11 vertices, 25 edges>
+gap> PlanarEmbedding(D);
+[ [ 3, 10, 5 ], [ 10, 8, 9 ], [ 4 ], [ 6 ], [ 11, 7 ], [ 7 ], [ 8 ], 
+  [  ], [ 11 ], [  ], [  ] ]
+gap> D := Digraph([[2, 4, 7, 9, 10], [1, 3, 4, 6, 9, 10], [6, 10], 
+> [2, 5, 8, 9], [1, 2, 3, 4, 6, 7, 9, 10], [3, 4, 5, 7, 9, 10], 
+> [3, 4, 5, 6, 9, 10], [3, 4, 5, 7, 9], [2, 3, 5, 6, 7, 8], [3, 5]]);
+<digraph with 10 vertices, 50 edges>
+gap> PlanarEmbedding(D);
+fail
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="OuterPlanarEmbedding">
+<ManSection>
+  <Attr Name="OuterPlanarEmbedding" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is an outer planar digraph, then 
+    <C>OuterPlanarEmbedding</C> returns the list of out-neighbours
+    of a subdigraph of <A>digraph</A> such that each vertex's
+    neighbors are given in clockwise order. If <A>digraph</A> is not outer planar,
+    then <K>fail</K> is returned. <P/>
+    
+    The directions and multiplicities of any edges in <A>digraph</A> are
+    ignored by <C>OuterPlanarEmbedding</C>.
+    <P/>
+
+    See also <Ref Prop="IsOuterPlanarDigraph"/>.
+    <P/>
+
+    This method uses the reference implementation in
+    &edge-addition-planarity-suite; by John Boyer of the algorithms described
+    in <Cite Key="Boyer2006aa"/>.
+
+<Example><![CDATA[
+gap> D := Digraph([[3, 5, 10], [8, 9, 10], [1, 4], [3, 6], 
+> [1, 7, 11], [4, 7], [6, 8], [2, 7], [2, 11], [1, 2], [5, 9]]);
+<digraph with 11 vertices, 25 edges>
+gap> OuterPlanarEmbedding(D);
+fail
+gap> D := Digraph([[2, 4, 7, 9, 10], [1, 3, 4, 6, 9, 10], [6, 10], 
+> [2, 5, 8, 9], [1, 2, 3, 4, 6, 7, 9, 10], [3, 4, 5, 7, 9, 10], 
+> [3, 4, 5, 6, 9, 10], [3, 4, 5, 7, 9], [2, 3, 5, 6, 7, 8], [3, 5]]);
+<digraph with 10 vertices, 50 edges>
+gap> OuterPlanarEmbedding(D);
+fail
+gap> OuterPlanarEmbedding(CompleteBipartiteDigraph(2, 2));
+[ [ 3, 4 ], [ 4, 3 ], [  ], [  ] ]
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="SubdigraphHomeomorphicToK">
+<ManSection>
+  <Attr Name="SubdigraphHomeomorphicToK23" Arg="digraph"/>
+  <Attr Name="SubdigraphHomeomorphicToK33" Arg="digraph"/>
+  <Attr Name="SubdigraphHomeomorphicToK4" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    These attributes return the list of out-neighbours of a subdigraph of
+    the digraph <A>digraph</A> which is homeomorphic to one of the following:
+    the complete bipartite graph with vertex sets of sizes <C>2</C> and
+    <C>3</C>; the complete bipartite graph with vertex sets of sizes <C>3</C>
+    and <C>3</C>; or the complete graph with <C>4</C> vertices. If
+    <A>digraph</A> has no such subdigraphs, then <K>fail</K> is returned. 
```

Comment:
Maybe emphasise the point that a digraph is planar/outer planar if and only if it has no such sub digraph?

Why does `SubdigraphHomeomorphicToK5` not exist?

Reply:
I've added a link to the doc for `IsPlanarDigraph` and `IsOuterPlanarDigraph`, rather than repeating the doc here again. 

> Why does SubdigraphHomeomorphicToK5 not exist?

Because it isn't implemented in the edge-addition-planarity-suite, and it isn't clear to me how to do this (at least in any good way). I did have a version of this PR where I had a function called `SubdigraphHomeomorphicTo(digraph, subdigraph)` but I abandoned it as too complicated in the first instance (it would only have worked for digraphs homeomorphic to K_4, K_{2, 3}, and K_{3, 3}, anyway.

Reply:
Fair enough, I'm surprised that it doesn't contain a K_5 thing, but that it does contain the other three, but then again I have no idea what the suite is doing!